src/service/fixed-layer.js

```diff
@@ -901,7 +1021,8 @@ class TransferLayerBody {
     }
     for (let i = 0; i < layerAttrs.length; i++) {
       const {name} = layerAttrs[i];
-      if (name === 'style' || body.hasAttribute(name)) {
+      if (name === 'style' || startsWith(name, 'i-amphtml-')
```

Comment:
Why was this added?

Reply:
Need to preserve the `i-amphtml-lightbox` attribute set when a lightbox is open.

Reply:
Make it explicit? As it is, any `i-amphtml-x` will never be removed once it's synced.